src/utils/RandomNumber.js

```diff
@@ -0,0 +1,13 @@
+const { Settings } = require('../Config');
+
+const RandomNumber = {
+  generateRandomNumber() {
```

Comment:
`generateRandomNumber`가 난수의 범위를 알고 있는 것이 아니라, 외부에서 그 범위를 지정해줄 수 있다면 더 사용성 높은 메서드가 됩니다.

Reply:
나중에 난수의 범위를 변경할 경우 사용성이 높아질 것 같습니다! 

```suggestion
  const RandomNumber = {
  generateRandomNumber(min, max) {
    return Math.floor(Math.random() * (max - min)) + min;
  },
};
```